ortools/java/com/google/ortools/sat/CpModel.java

```diff
@@ -966,6 +966,13 @@ public void addHint(IntVar var, long value) {
     modelBuilder.getSolutionHintBuilder().addValues(value);
   }
 
+  /** Adds variable as assumption */
+  public void addAssumptions(Literal[] literals) {
+    for (Literal lit : literals) {
+      modelBuilder.addAssumptions(lit.getIndex());
+    }
+  }
```

Comment:
I would add both the single var, and the var[] API.

Reply:
Okey!

Reply:
Overload or two methods (without the s)?